policyengine_us/variables/gov/states/ar/tax/income/deductions/itemized/ar_itemized_deductions.py

```diff
@@ -0,0 +1,69 @@
+from policyengine_us.model_api import *
+
+
+class ar_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=21"
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        agi = tax_unit("adjusted_gross_income", period)
+        spouse_agi = tax_unit("spouse_separate_adjusted_gross_income", period)
+
+        # Less salt deduction
+        p = parameters(period).gov.states.ar.tax.income.deductions.itemized
+        less_salt_deds = tax_unit("itemized_deductions_less_salt", period)
```

Comment:
def formula(tax_unit, period, parameters):
        p = parameters(period).gov.irs.deductions
        deductions = [
            deduction
            for deduction in p.itemized_deductions
            if deduction not in ["salt_deduction", "medical_expense_deduction",  ]
        ]
        itemized =  add(tax_unit, period, deductions)

Reply:
Do we need to add an input for the this sumed itemized deductions in the test, if we do, then what't the input's name? Thank you!

Reply:
I'm wondering that if we need to add inputs for this sumed itemized deductions in the test file? If we need, then do we need to add all the deductions in the p.itemized_deductions, like setting all the following deductions as inputs in the test file?
    - charitable_deduction
    - interest_deduction
    - salt_deduction
    - medical_expense_deduction
    - casualty_loss_deduction

Reply:
all of them but - salt_deduction

Reply:
Got it! Thank you!